Sourcery/Generating/Template/Stencil/StencilTemplate.swift

```diff
@@ -30,6 +30,13 @@ final class StencilTemplate: Stencil.Template, Template {
     private static func sourceryEnvironment() -> Stencil.Environment {
         let ext = Stencil.Extension()
         ext.registerFilter("upperFirst", filter: Filter<String>.make({ $0.upperFirst() }))
+        ext.registerFilterWithTwoArguments("replace", filter: { (s1: Any?, s2: Any, s3: Any) -> Any? in
+            if let s1 = s1 as? String, let s2 = s2 as? String, let s3 = s3 as? String {
+                return s1.replacingOccurrences(of: s2, with: s3)
+            }
+            return s1
```

Comment:
but only parameters, we don't throw on the value type (and careful with optionality)